### ISSUE TITLE
All pages in the DSpace User Interface should have automated accessibility tests

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#2780]
## Description

As of #2711, the following User Interface pages _have automated accessibility scanning_ (via Deque's Axe tool) enabled:
* All current, anonymous-access pages (at least that I'm aware of)
* Submission / MyDSpace / Workflow pages
* Community/Collection/Item Edit pages (all tabs)
* Login, new user registration & user menu
* Admin sidebar 

**The following pages are believed to be missing accessibility scanning:**
* All admin pages linked of Admin Sidebar not listed above.
* Any additional features (or new features) not listed above

## Proposed solution

All DSpace user interface pages / modals should have an automatic accessibility scan enabled.  
* This requires building a Cypress e2e test script for that page (if one doesn't exist), loading the page and running the `testA11y()` function on the page similar to the examples in #2711
* If the `testA11y()` script fails (because accessibility issues exist on the page), then either the issues should be fixed *or* a temporary exclusion can be added to the `testA11y()` script until those issues can be fixed.  For examples of exclusions see [`submission.cy.ts](https://github.com/DSpace/dspace-angular/blob/main/cypress/e2e/submission.cy.ts)`

_This ticket need not be solved by a single PR._  In fact, it may be easier to implement these improvements over several PRs.  Therefore, multiple people can claim this ticket *provided they are clear about which pages they are working on.*

## Related work
Related to the work in #2711, which builds on some earlier automatic accessibility scanning.

**Comment by tdonohue** [comment]
@atarix83 : I think it's perfectly OK to *only create* the accessibility validation for the new e2e tests.  That also has the benefit of creating a "basic scaffolding" for all pages to have more thorough e2e tests in the future (as part of a future effort).

Assigned to you and adding estimate details as a label.
